config/betterquesting/DefaultQuests/Quests/CoinsCoinsCoins-AAAAAAAAAAAAAAAAAAAAEA==/TNTandiTNT-beN48jZMRCqH2QGlTH7nTg==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Want some TNT ot iTNT? Buy some!"
+      "desc:8": "Want some TNT or iTNT? Buy some!"
```

Comment:
Industrial TNT maybe given that is the actual in-game name.

Reply:
I've always referred to it as iTNT and seen it referred to as such so unsure what would be more fitting personally

Reply:
seems rocket quests and a couple others refer to it as iTNT, while the rest (only about 5 quests) call it industrial TNT

one of the quests that call it industrial TNT however is the quest titled iTNT...

Reply:
staying with iTNT
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/38961580/df85cf48-8fa5-4b42-935e-beb8ba5b67d8)